scripts/presubmit-tests.sh

```diff
@@ -61,7 +61,7 @@ function pr_only_contains() {
 function list_changed_files() {
   local file="${WORK_DIR}/changed_files"
   if [[ ! -f ${file} ]]; then
-    githubhelper -list-changed-files > ${file}
+    git --no-pager diff --name-only ${PULL_BASE_SHA}..${PULL_SHA} > ${file}
```

Comment:
The GitHub API might do something more than this, but it's worth skipping the extra roundtrip :)